account_payment_term_extension/models/account_payment_term.py

```diff
@@ -1,17 +1,50 @@
 # -*- coding: utf-8 -*-
-# © 2013-2016 Camptocamp SA (Yannick Vaucher)
-# © 2004-2016 Odoo S.A. (www.odoo.com)
-# © 2015-2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# Copyright 2013-2016 Camptocamp SA (Yannick Vaucher)
+# Copyright 2004-2016 Odoo S.A. (www.odoo.com)
+# Copyright 2015-2016 Akretion
+# (Alexis de Lattre <alexis.delattre@akretion.com>)
+# Copyright 2018 Simone Rubino - Agile Business Group
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 
 from dateutil.relativedelta import relativedelta
 from odoo import models, fields, api, exceptions, _
+from odoo.exceptions import ValidationError
 from odoo.tools.float_utils import float_is_zero, float_round
 import odoo.addons.decimal_precision as dp
 import calendar
 
 
+class AccountPaymentTermHoliday(models.Model):
+    _name = 'account.payment.term.holiday'
+
+    payment_id = fields.Many2one(comodel_name='account.payment.term')
+    holiday = fields.Date(required=True)
```

Comment:
This is date_start of the holiday? we should rename it, for better understanding of the code

Reply:
Hi, thanks for the review. 
This has actually been thought for one-day holidays, to postpone payments that should be on one specific day (25th December for instance).
In order to manage holiday-ranges it would require a little more implementation during the `holiday_check` that I can't perform right now, you're welcome to PR my branch anyway.